src/utils/scheduleTasks.js

```diff
@@ -47,4 +55,23 @@ const deleteExpiredMoments = async () => {
   }
 };
 
+const sendLogMessagesToDiscord = async () => {
+  const logFilePath = path.join(__dirname, '..', 'log', 'requests.log');
+  const logMessages = fs.readFileSync(logFilePath, 'utf8').split('\n');
+  logMessages.pop();
+
+  for (const logMessage of logMessages) {
+    try {
+      const channel = await client.channels.fetch(discordChannelId);
+      if (channel) {
+        await channel.send(logMessage);
+      }
+    } catch (error) {
+      console.error('Failed to send message to Discord:', error);
+    }
+  }
+
+  fs.truncate(logFilePath, 0, () => {});
+};
```

Comment:
thay vì split về mảng thì em nên push toàn bộ nội dung log cho nó giảm tải server